### PR TITLE
feat: add active and disabled object tables to settings

### LINKED ISSUE #1799: Display collapsible active objects table
## Context

See #1762

### LINKED ISSUE #1800: Display collapsible disabled objects table
## Context

See #1762

### PR DESCRIPTION
**Pull request by thaisguigon**
Closes #1799, Closes #1800

### CODE REVIEW

File: front/src/modules/ui/table/components/TableCell.tsx [hunk 1]
```diff
@@ -1,18 +1,32 @@
...
-  <StyledTableCell>
-    <StyledTableCellContent>{children}</StyledTableCellContent>
-  </StyledTableCell>
-);
+export const TableCell = ({ children, className }: TableCellProps) => {
+  return (
+    <StyledTableCell
+      className={className}
+      isNumberCell={
+        typeof children === 'number' ||
```
Comment (lucasbordeau) [inline front/src/modules/ui/table/components/TableCell.tsx:25]: Could we use a `isNumber()` typeguard ?

**Review by charlesBochet — APPROVED** [review]

File: front/src/modules/ui/table/components/TableCell.tsx [hunk 2]
```diff
@@ -1,18 +1,32 @@
...
-export const TableCell = ({ children }: PropsWithChildren) => (
-  <StyledTableCell>
-    <StyledTableCellContent>{children}</StyledTableCellContent>
-  </StyledTableCell>
-);
+export const TableCell = ({ children, className }: TableCellProps) => {
+  return (
+    <StyledTableCell
+      className={className}
+      isNumberCell={
```
[the remark this answers is not in the document]
Comment (charlesBochet) [reply front/src/modules/ui/table/components/TableCell.tsx:24]: maybe an alignment = 'right' | 'left' as a prop

[the remark this answers is not in the document]
Comment (thaisguigon) [reply front/src/modules/ui/table/components/TableCell.tsx]: Done!

File: front/src/modules/ui/table/components/TableCell.tsx (hunk 1 above)
Comment (thaisguigon) [reply front/src/modules/ui/table/components/TableCell.tsx]: I implemented an `align` prop as suggested by @charlesBochet (see https://github.com/twentyhq/twenty/pull/1885#discussion_r1347665751), therefore this check is not needed anymore.

**Review by lucasbordeau — APPROVED** [review]

**Pull request merged**